src/java/org/apache/cassandra/service/StorageService.java

```diff
@@ -1415,6 +1416,17 @@ public void setCompactionThroughputMbPerSec(int value)
                     value, oldValue);
     }
 
+    public Map<String, String> getCurrentCompactionThroughput()
+    {
+        HashMap<String, String> result = new LinkedHashMap<>();
+        Meter rate = CompactionManager.instance.getCompactionThroughput();
+        double mb = 1024.0 * 1024.0;
```

Comment:
we have `org.apache.cassandra.io.util.FileUtils.ONE_MIB` to use. You would just make double from it if necessary.